hack/all-in-one/Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN export VERSION=$(curl -s https://api.github.com/repos/redhat-et/microshift/r
 
 COPY unit /usr/lib/systemd/system/microshift.service
 COPY kubelet-cgroups.conf /etc/systemd/system.conf.d/kubelet-cgroups.conf
+COPY crio-bridge.conf /etc/cni/net.d/100-crio-bridge.conf
```

Comment:
can you also add the [conmon change](https://github.com/redhat-et/microshift/blob/main/install.sh#L153) too? Since this is the ubi image

Reply:
The conmon change was in crio.conf (not the CNI). Is that also not working in the AIO image?